### ISSUE TITLE
jsonSerialize ignore optional attributes

### ISSUE THREAD

**Issue by eav93** [php-telegram-bot/core#1303]
### 🐞 Bug Report

```
$data = [
            'reply_markup' => (new Keyboard(
                [
                    'One',
                    'Two'
                ],
                [
                    'Three',
                    'Four',
                ]
            ))->setResizeKeyboard(true)
        ];
```
Entity::jsonSerialize() return only raw_data but does not return optional attributes: resize_keyboard, one_time_keyboard, etc...

**Comment by TiiFuchs** [comment]
Can confirm. I'm working on a fix.

**Comment by TiiFuchs** [comment]
This bug has nothing to do with the introduction of jsonSerialize. It was there before.
I fixed it with PR #1304 though. Should be in the next release. Trying to release later today.
